### DISCUSSION TITLE
Ομοιομορφία στα CVs

### DISCUSSION THREAD

**Discussion by ghost** [sdy60-2022/.github#18] [General]
Θα έχουμε ομοιομορφία στα CVs ή μπορούμε να πειράξουμε τα CSS;

**Comment by stdi** [comment]
Έχεις δίκιο για το ερώτημα, μήπως να το ρωτήσουμε το Σάββατο στη μίνι συνάντηση;

**Reply by NikosAndreadakis** [reply]
Επίσης να ρωτήσουμε και αν το bootstrap που έχουμε σαν template είναι αποδεκτό.

**Reply by atsiano81** [reply]
Δε νομίζω να  υπάρξει θέμα με το bootstrap open source είναι έτσι και αλλιώς και mainstream

**Reply by stdi** [reply]
Το πρόβλημα που θα μπορούσε να υπάρξει είναι να φαίνεται σαν να έχουν κάνει όλοι το ίδιο πράγμα.

**Reply by ghost** [reply]
Πολύ σωστό αυτό που σκέφτηκες Αλέξανδρε. Οπότε θα πρότεινα να κάνουμε και αλλαγές στα stylesheets όσοι θέλουμε.
